src/Apps/Artist/Routes/Overview/Components/ZeroState.tsx

```diff
@@ -1,8 +1,9 @@
 import { Link, Message } from "@artsy/palette"
-import { AnalyticsSchema, useSystemContext } from "Artsy"
```

Comment:
@eessex - we should avoid `import * as SchemaV2` and stick to the named export format as seen here